RELEASE_PROCESS.md

```diff
@@ -8,33 +7,25 @@ provisions of the license provided by the LICENSE file in this repository.
 This document describes the process for creating a new version of the
 signal specification.
 
-The process, driven by the GENIVI Remote Vehicle Interaction Expert
-Group (RVI EG), is aimed at being lightweight and carried out in public, giving both
-GENIVI members and non-members a say in the creation of a new version.
+The process, driven by GENIVI, is aimed at being lightweight and carried
```

Comment:
I might think to consider adding:
"...by GENIVI, with significant input from the W3C Automotive WG, is aimed at...
It's not the only stakeholder, and other GENIVI projects are increasingly working with this but lately a lot of drive has come from the discussions held in W3C context.

Reply:
This is a comment for others to consider first and discuss.  My second comment is a "required" change from me.

Reply:
@gunnarx: I thought about a similar thing at the beginning, but decided against it, as discussion is taking place in different places (e.g. layers discussion started in the android group, etc.). The important thing is, that it's open and the information is collected and linked here. But I'm not strongly against it.

Thanks for spotting the typo. I fixed it.

Reply:
I welcome additional wording to reflect the regular coordination between GENIVI and W3C on this topic. As this is GENIVI's repo, the decision is theirs.

Reply:
@tguild: added it